ercc/registry/registry_test.go

```diff
@@ -48,7 +55,7 @@ type identityEvaluator interface {
 var (
 	mrenclave              = `98aed61c91f258a37c68ed4943297695647ec7bbe6008cc111b0a12650ebeb91`
 	channelId              = "MY_TEST_CHANNEL"
-	chaincodeId            = "SOME_CHAINCODE_ID"
+	chaincodeId            = "SOME_CHAINCODE_PKG_ID"
```

Comment:
I think the var `chaincodeId` really is the chaincode id (ie "ercc" or "ecc") not the pkg? Maybe I am wrong?

Reply:
you are wrong: "// CCID should match chaincode's package name on peer" quote from https://godoc.org/github.com/hyperledger/fabric-chaincode-go/shim#ChaincodeServer